rcp/org.testeditor.rcp4.views.teststepselection/src/org/testeditor/rcp4/views/teststepselector/TestStepSelectorExecutionListener.xtend

```diff
@@ -0,0 +1,39 @@
+package org.testeditor.rcp4.views.teststepselector
+
+import org.eclipse.core.commands.IExecutionListener
+import org.eclipse.core.commands.NotHandledException
+import org.eclipse.core.commands.ExecutionException
+import org.eclipse.core.commands.ExecutionEvent
+import org.eclipse.ui.IWorkbenchCommandConstants
+import org.testeditor.aml.dsl.ui.internal.DslActivator
+import org.eclipse.ui.PlatformUI
+import javax.inject.Inject
+import org.eclipse.e4.core.services.events.IEventBroker
+import org.eclipse.e4.core.di.annotations.Creatable
+
+@Creatable
+class TestStepSelectorExecutionListener implements IExecutionListener {
+
+	@Inject IEventBroker broker
+	
+	override notHandled(String commandId, NotHandledException exception) {
+	}
+
+	override postExecuteFailure(String commandId, ExecutionException exception) {
+	}
+
+	override postExecuteSuccess(String commandId, Object returnValue) {
+		if (IWorkbenchCommandConstants.FILE_SAVE.equals(commandId)) {
```

Comment:
In Xtend you can use `commandId == File_SAVE` (which transforms to an equals check)